consumer/start_consumer.sh

```diff
@@ -0,0 +1,315 @@
+#!/bin/bash
+set -eux
+
+SOVEREIGN_HOME="$HOME/.sovereign"
```

Comment:
Great stuff! This file is not auto generated right? 

Also, did you base this file off existing setup shell scripts such as `tests/integration/testnet-scripts`?

Reply:
It's built in custom, didn't have chance to check testnet-scripts directory.

Reply:
The scripts on testnet-scripts directory has a lot of difference with the newly built script and I think base fines on testnet-scripts would be hard to use for now.

Reply:
It should be fine to leave these shell files as-is. However, it would be great to have these tests run as a part of the CI/CD process in a docker container. We already have such a system in the `/tests/integration` directory. 

I'd suggest taking a look at that directory and seeing if you could tie in these tests into what we define as [integration tests](https://github.com/cosmos/interchain-security/blob/main/docs/testing.md#integration-tests), or at least pattern matching some of the automation we've setup there

Reply:
One of my concern is that the block heights and etc could be a bit of different per machine. This was working fine on my machine FYI.